spec/helpers/locale_helper_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.describe LocaleHelper do
+  include LocaleHelper
+
+  describe "#translations_for_nav" do
+    it "returns translations in a suitable format for the translation nav component" do
+      translations = [
+        { "locale" => "en", "base_path" => "/" },
+        { "locale" => "cy", "base_path" => "/cy" },
+      ]
+
+      expect(translations_for_nav(translations)).to eq(
+        [
+          {
+            active: true,
+            locale: "en",
+            base_path: "/",
+            text: "English",
+          },
+          {
+            locale: "cy",
+            base_path: "/cy",
+            text: "Cymraeg",
+          },
+        ],
+      )
+    end
+  end
+
+  describe "#t_locale_fallback" do
+    it "returns nil if a value of the key is present" do
+      expect(t_locale_fallback("formats.take_part", count: 1)).to be_nil
```

Comment:
So `t_locale_fallback` method returns the "fallback", i.e. the English translation, if the language key doesn't exist? I feel like the language keys _should_ exist, and they default to English by default if there's no value.

Actually, I've just looked at the [commit message](https://github.com/alphagov/government-frontend/commit/c409f0f01404b7ccb1792f330ce415aa11314bec) from when the original method was added and it's existence makes sense.

Reply:
(Positive, so marking as resolved)